regression/goto-instrument/remove-function-body1/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+main.c
+--remove-function-body foo --remove-function-body bar
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION SUCCESSFUL$
```

Comment:
I appreciate that it is hard to actually test anything from the output, but it might be good if we could run `--show-goto-functions` on the output and check that bar is really removed? 

Alternatively, if you grab #874, it might be relatively easy to add some unit tests for this.